lib/themes/dosomething/paraneue_dosomething/scss/patterns/_button.scss

```diff
@@ -1,14 +1,73 @@
+%animated {
```

Comment:
I _thiiiiink_ you should be able to use Forge's existing [`%animated`](https://github.com/DoSomething/forge/blob/dc8d2a60ce5b09c448007a2ad5b1d353e1cd901a/scss/_base/_animations.scss#L1-L5) placeholder selector here.